documents-git/includes/providers/class-base-loader.php

```diff
@@ -121,6 +121,10 @@ public function doJupyter($sc_attrs)
      */
     public function doMarkdown($sc_attrs)
     {
+        if ($cached_response = $this->get_cached_content(md5($sc_attrs), 'markdown')) {
```

Comment:
When I was developing this code, I've used `sc_attrs['url']`, but, in some cases, this can be a problem when we need to force re-cache content. Using all properties made this easier, ie, changing TTL only.

About `md5()`, I really didn't know that. Considering the comment above, which do you think is better way?

* Keep all sc_attrs and serialize them?
* Use only URL and move md5 digest to get/set_content_cache?

Reply:
> in some cases, this can be a problem when we need to force re-cache content.

Ah ok, that makes sense. Sure, I don't really care actually, we can serialize all of the shortcode attributes. I guess you can still move it into the getter/setter so there's only 2 instances of md5().